scripts/completion/dvc.bash

```diff
@@ -10,33 +10,54 @@
 
 _dvc_commands='init destroy add import checkout run pull push fetch \
               status repro remove move unprotect gc config remote metrics \
-              install root lock unlock pipeline commit'
+              install root lock unlock pipeline commit cache cache_dir diff \
+              config index metrics_add metrics_modify metrics_remove move \
```

Comment:
:thinking: how does `metrics_show` work? is it `dvc metrics s[tab]` -> `dvc metrics show` ?

Reply:
I don't know. I should upgrade my dvc version

Reply:
@Naba7 I think @mroutis is asking how the completion works, not the `dvc metrics show` command itself.

Reply:
It shows all the files and directories whether it is dvc or not.

Reply:
@Naba7 Right, but `dvc metrics s[tab]` should complete into `dvc metrics show` command.

Reply:
@efiop I have to do a double ``tab`` to get the files or directories starting with ``s`` in their name like ``dvc metrics [tab][tab] ``. It doesn't autocomplete ``dvc metrics show``